### ISSUE TITLE
Add option to ignore MD013 for reference links

### ISSUE THREAD

**Issue by benjamb** [joshrotenberg/mdbook-lint#407]
If using reference links and the combination of the link label and the link destination is greater than the configured line length, MD013 is triggered. It would be good if there were an option to disable MD013 in such cases.

Example reference link that triggers the issue with line length at 80:
```
[this-triggers-md013-when-long-link-and-label]: https://github.com/mdbook-lint/issues
```

**Comment by benjamb** [comment]
@joshrotenberg Thanks!
